addons/website/static/src/builder/plugins/form/form_option.xml

```diff
@@ -230,6 +230,60 @@
                 records="state.valueList.availableRecords" />
         </div>
     </BuilderRow>
+    <BuilderRow label.translate="Character Limit" t-if="domStateCurrentFieldInput.isTextArea or ['text'].includes(domStateCurrentFieldInput.type)">
+        <BuilderCheckbox
```

Comment:
Strange but character limit is shown in date type field
<img width="1597" height="559" alt="image" src="https://github.com/user-attachments/assets/a3844b29-6303-4216-8e47-ca655202c721" />